### ISSUE TITLE
the load_checkpoints func is confused

### ISSUE THREAD

**Issue by gyula-coder** [tangjiapeng/DiffuScene#35]
initially, the load_checkpoints is imported at the line 20 of generate_objautoencoder by `from utils import yield_forever, load_checkpoints, save_checkpoints` but there is no load_checkpoints. After I change the utils to training_utils, the load_checkpoints is imported correctly.

But when I run the generate_objautoencoder.py, the check of optimizer's path, opt_path, before model.load_state_dict in load_checkpoints will failed and result early return. So autoencoder's parameter is still initial !

**Comment by tangjiapeng** [comment]
Hi have you debugged this issu ? If not yet, can you provide some error information ?

**Comment by gyula-coder** [comment]
yes, I modified the load_checkpoints and let it to only load the autoencoder's checkpoint. It works.
I find there are lot of mismatch betweeen code and path, which makes running code is hard when preprocessing the data. 
What‘s more, the preprocessed data is unavailable because of it's predefine path added when preprocessing, which is used when retrive_objfeats is True.

At last, I change lots of path to make the generate_diffusion.py running.

**Comment by tangjiapeng** [comment]
yes, you always need to change the paths.
